node-tests/fixtures/template.tagname

```diff
@@ -0,0 +1 @@
+MyCustomElement
```

Comment:
Is this file used? On initial review, the tests seem to write this file manually via `input.write`...

Reply:
Yes, it's used to populate the initial version of the file  in the `beforeEach` above:

```js
input.write(fixturify.readSync(`${__dirname}/fixtures`));
```